docs/podman-container-restore.1.md

```diff
@@ -40,6 +40,11 @@ If the checkpoint image does not contain established TCP connections this
 option is ignored. Defaults to not restoring containers with established TCP
 connections.
 
+**--import, -i**
+
+Import a checkpoint tar.gz file, which was exported by Podman. This can be used
+to import a checkpointed container from another host.
+
```

Comment:
Should we add that "It is not necessary to specify a container when restoring from export."?